server/controllers/wazuh-utils.ts

```diff
@@ -126,4 +127,47 @@ export class WazuhUtilsCtrl {
       return ErrorResponse(error.message || error, 3036, 500, response);
     }
   }
+
+
+    /**
+   * Returns Wazuh frontend logs
+   * @param {Object} context 
+   * @param {Object} request
+   * @param {Object} response
+   * @returns {Array<String>} app logs or ErrorResponse
+   */
+    async getFrontendLogs(context: RequestHandlerContext, request: KibanaRequest, response: KibanaResponseFactory) {
+      try {
+        const lastLogs = await read(
+          WAZUH_FRONTEND_LOGS_RAW_PATH,
+          50
+        );
+        const spliterLog = lastLogs.split('\n');
+        return spliterLog && Array.isArray(spliterLog)
+          ? response.ok({
+            body: {
+              error: 0,
+              lastLogs: spliterLog.filter(
+                item => typeof item === 'string' && item.length
+              )
+            }
+          })
+          : response.ok({ error: 0, lastLogs: [] });
+      } catch (error) {
+        return ErrorResponse(error.message || error, 3036, 500, response);
+      }
+    }
+
+    async updateFrontendLogs(context: RequestHandlerContext,request: KibanaRequest,response: KibanaResponseFactory) {
+      try {
+        await addFrontendLog(request.body.location, request.body.message, request.body.level);
+        return response.ok({
+          body: {
+            message: 'Log has been added',
+          },
+        });
+      } catch (error) {
+        return ErrorResponse(error.message || error, 3021, 500, response);
+      }
+    }
```

Comment:
move this endpoints controller to a new file may be `server/controllers/wazuh-utils/ui-logs.controller.ts` 
also, move `wazuh-utils.ts` into this new folder and export both from an `index.ts`